custom-elements/reactions/HTMLHyperlinkElementUtils.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<title>Custom Elements: CEReactions on HTMLHyperlinkElementUtils interface</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<meta name="assert" content="href, protocol, username, password, host, hostname,
+  port, pathname, search, hash of HTMLHyperlinkElementUtils interface must have
+  CEReactions">
+<meta name="help" content="https://html.spec.whatwg.org/#api-for-a-and-area-elements">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="../resources/custom-elements-helpers.js"></script>
+<script src="./resources/reactions.js"></script>
+
+<body>
+<script>
+
+function getParentElement() {
+  return document.body;
+}
+
+function setAttributes(instance) {
+  instance.setAttribute('href', 'http://w3c-test.org');
+}
+
+testReflectAttribute(
+  'href', 'href', 'http://w3c-test.org',
+  'https://w3c-test.org', 'href on HTMLHyperlinkElementUtils in a', 'a',
+  HTMLAnchorElement
+);
+testReflectAttributeWithDependentAttributes(
+  'protocol', 'protocol', 'https:',
+  'http:', 'protocol on HTMLHyperlinkElementUtils in a', 'a',
+  getParentElement, instance => setAttributes(instance), HTMLAnchorElement
```

Comment:
Most of these tests fail, in both Firefox and Chrome. See e.g. https://wpt.fyi/results/custom-elements/reactions/HTMLHyperlinkElementUtils.html?diff&filter=ADC&run_id=6318984831959040&run_id=5147145954918400 . I suspect something is wrong with the tests.

Maybe this is connected to my question of why you are using testReflectAttributeWithDependentAttributes instead of testReflectAttribute.